scripts/deploy/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "2"
+services:
+  imagespace-mongo:
+    image: mongo:3.0
+    networks:
+      - imagespace-network
+  imagespace-solr:
+    image: solr:6.0
+    networks:
+      - imagespace-network
+    entrypoint:
+      - docker-entrypoint.sh
+      - solr-precreate
+      - imagespace
+  imagespace-girder:
+    image: kitware/image_space:latest
```

Comment:
why does this depend on something specific to kitware? Shouldn't it be memex-explorer/image_space:latest?